app-shells/fzf/fzf-0.21.0.ebuild

```diff
@@ -0,0 +1,124 @@
+	"golang.org/x/sys v0.0.0-20190412213103-97732733099d/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs="
+	"golang.org/x/sys v0.0.0-20190626150813-e07cf5db2756 h1:9nuHUbU8dRnRRfj9KjWUVrJeoexdbeMjttk6Oh1rD10="
+	"golang.org/x/sys v0.0.0-20190626150813-e07cf5db2756/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs="
+	"golang.org/x/sys v0.0.0-20200116001909-b77594299b42/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs="
+	"golang.org/x/sys v0.0.0-20200202164722-d101bd2416d5 h1:LfCXLvNmTYH9kEmVgqbnsWfruoXZIrh4YBgqVHtDvw0="
+	"golang.org/x/sys v0.0.0-20200202164722-d101bd2416d5/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs="
+	"golang.org/x/text v0.3.0 h1:g61tztE5qeGQ89tm6NTjjM9VPIm088od1l6aSorWRWg="
+	"golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ="
+	"golang.org/x/text v0.3.2 h1:tW2bmiBqwgJj/UpqtC8EpXEZVYOwU0yG4iWbprSVAcs="
+	"golang.org/x/text v0.3.2/go.mod h1:bEr9sfX3Q8Zfm5fL9x+3itogRgK3+ptLWKqgva+5dAk="
+	"golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ="
+	"golang.org/x/tools v0.0.0-20191011211836-4c025a95b26e h1:1o2bDs9pCd2xFhdwqJTrCIswAeEsn4h/PCNelWpfcsI="
+	"golang.org/x/tools v0.0.0-20191011211836-4c025a95b26e/go.mod h1:b+2E5dAYhXwXZwtnZ6UAqBI28+e2cm9otk0dWdXHAEo="
+	"golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7/go.mod h1:I/5z698sn9Ka8TeJc9MKroUUfqBBauWjQqLJ2OPfmY0="
+)
+go-module_set_globals
+
+EGO_PN="github.com/junegunn/${PN}"
+SRC_URI="https://${EGO_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+			${EGO_SUM_SRC_URI}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="fish-completion neovim tmux vim zsh-completion"
+
+DEPEND=""
+RDEPEND="
+	${DEPEND}
+	fish-completion? ( app-shells/fish )
+	tmux? ( app-misc/tmux )
+	neovim? ( app-editors/neovim )
+	vim? ( app-editors/vim )
+	zsh-completion? ( app-shells/zsh )
+"
+BDEPEND=""
+
+S="${WORKDIR}/${P/_/-}"
+
+src_install() {
+	default
+	# Install fzf binary
+	dobin bin/*
+
+	# Install fish key-bindings
+	if use fish-completion; then
+		insinto /usr/share/fish/functions/
+		newins shell/key-bindings.fish fzf_key_bindings.fish
+	fi
+
+	# Install Neovim plugin
+	if use neovim; then
+		insinto /usr/share/nvim/runtime/plugin
+		doins plugin/${PN}.vim
+	fi
+
+	# Install TMUX utils
+	if use tmux; then
+		dobin bin/${PN}-tmux
+		doman man/man1/${PN}-tmux.1
+	fi
+
+	# Install VIM plugin
+	if use vim; then
+		insinto /usr/share/vim/vimfiles/plugin
+		doins plugin/${PN}.vim
+	fi
+
+	# Install zsh completion file and key-bindings
+	if use zsh-completion; then
+		insinto /usr/share/zsh/site-functions
+		newins shell/completion.zsh _${PN}
+		insinto /usr/share/zsh/site-contrib/
+		newins shell/key-bindings.zsh ${PN}.zsh
+	fi
+
+	# Install bash completion file and key-bindings
+	newbashcomp shell/completion.bash ${PN}
+#	mkdir -p "${ED}$(get_bashcompdir)/"
+#	cp shell/completion.bash "${ED}$(get_bashcompdir)/${PN}"
```

Comment:
Debugging lines should be removed when it works.

Reply:
Removed